src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -13,16 +13,65 @@ namespace System.Net.Security
     internal sealed class SafeFreeNegoCredentials : SafeFreeCredentials
     {
         private SafeGssCredHandle _credential;
+        private readonly bool _isNtlm;
+        private readonly string _userName;
+        private readonly bool _isDefault;
 
         public SafeGssCredHandle GssCredential
         {
             get { return _credential; }
         }
 
-        public SafeFreeNegoCredentials(string username, string password, string domain) : base(IntPtr.Zero, true)
+        public bool IsNtlm
```

Comment:
isNtlmOnly and isNtlm can be considered as same thing .In fact we can rename isNtlmOnly to isNtlm , ultimately both represents only ntlm protocol to be used .. not spnego.. 
but IsntlmUsed  represents what it negotiated internally even if spnego is passed ( represents fallback ) .